### ISSUE TITLE
Loss calculation in getPostError

### ISSUE THREAD

**Issue by Jetafull** [FrankPortman/bayesAB#40]
I think to calculate the expected loss for choosing A over B, in this case, is to calculate the percentage change of A over B, when B is better than A. Should the formula be: `loss = -f(A_samples, B_samples)` instead of `loss = f(B_samples, A_samples)`? 

https://github.com/FrankPortman/bayesAB/blob/9db6f4629f93bb7b08e3b19950804c9643462708/R/util.R#L19
